sky/utils/command_runner.py

```diff
@@ -41,6 +41,8 @@
 # The git exclude file to support.
 GIT_EXCLUDE = '.git/info/exclude'
 RSYNC_EXCLUDE_OPTION = '--exclude-from={}'
+# Owner and group metadata is not needed.
+RSYNC_NO_OWNER_NO_GROUP_OPTION = '--no-owner --no-group'
```

Comment:
Should we only add this for the download, i.e. preserve it for the upload to remote cluster?